views/home.dt

```diff
@@ -1,117 +1,75 @@
 extends layout
 
 block title
-	- import dubregistry.viewutils;
-	- import dubregistry.web;
-	- import vibe.data.json;
-	- import std.algorithm.comparison : min;
-	- import std.algorithm.iteration : map;
-	- import std.string : replace, split;
-	- auto title = "Find, Use and Share DUB Packages";
-	script(type="application/javascript", src="scripts/home.js")
-	script(type="application/javascript").
-		window.categories = #{serializeToJson(info.categories).toString()};
+	- auto title = "DUB Package Manager";
 
 block body
-	- auto category = req.query.get("category", null);
-	- auto sort_mode = req.query.get("sort", "updated");
+	h1 DUB
+	h2 D&apos;s Official Package Manager
+	p.
+		DUB allows you to easily share your libraries and applications with others
+		and reuse code components or run applications made by others.
 
-	- string makeHomeURL(string sort, string category, ulong skip, ulong limit) {
-	-   import std.conv : to;
-	-   return "?sort=" ~ sort ~ "&category=" ~ category ~ "&skip=" ~ skip.to!string ~ "&limit=" ~ limit.to!string;
-	- }
+	h2 Getting Started
+	p Creating projects with dub is easy. To create a new project simply run
+	pre>< $ dub init hello_world
+	p You will be able to customize your project in the init wizard:
+	pre><.
+		Package recipe format (sdl/json) [json]: 
+		Name [hello_world]: 
+		Description [A minimal D application.]: 
+		Author name [user]: 
+		License [proprietary]: 
+		Copyright string [Copyright © 2017, user]: 
+		Add dependency (leave empty to skip) []: 
+		Successfully created an empty project in '/home/user/hello_world'.
+		Package successfully created in hello_world
+	p.
+		If you want, you can change values here or just press enter to keep the
+		suggested value.
+	pre><.
+		$ cd hello_world/
+		$ tree .
+		.
+		├── dub.json
+		└── source
+				└── app.d
 
-	- auto mirror = req.params.get("mirror", "");
-	- if (mirror.length)
-		p This is a mirror of #[a(href=mirror)= mirror].
-	- else
-		p Welcome to DUB, the D package registry. The following list shows all available packages:
+		1 directory, 2 files
+	p.
+		The #[code dub.json] contains the metadata about the package, you can edit
+		it to adjust information about your package or add or remove dependencies.
+	a(href="package-format")
+		button Learn More
+	p.
```

Comment:
That's nice for beginners but the site should also cater to regular users. I'd rather see a search box on the main page.
I'd be in favor of a call to action `Getting Started` button near the search box that links to getting started, e.g. similar to "Install RubyGems".
![image](https://user-images.githubusercontent.com/288976/35439566-94fe58d2-029b-11e8-9591-2f759e5629b9.png)
https://rubygems.org/

<hr>

We also want to show 3 lists à 5 or 10 entries below the search part.

Trending | Highest score | Random or Latest updates
------------|-------------------|----------------------------------
[requests](https://code.dlang.org/packages/requests) | [vibe-d](https://code.dlang.org/packages/vibe-d) | [qrcode](https://code.dlang.org/packages/qrcode)
[dlangui](https://code.dlang.org/packages/dlangui) | [dub](https://code.dlang.org/packages/dub) | [collie](https://code.dlang.org/packages/collie)
[mir-algorithm](https://code.dlang.org/packages/mir-algorithm) | [emsi_containers](https://code.dlang.org/packages/emsi_containers) | [kiss](https://code.dlang.org/packages/kiss)
[fluent-asserts](https://code.dlang.org/packages/fluent-asserts) | [unit-threaded](https://code.dlang.org/packages/unit-threaded) | [tsv-utils-dlang](https://code.dlang.org/packages/tsv-utils-dlang)
[vectorflow](https://code.dlang.org/packages/vectorflow) | [derelict-sdl2](https://code.dlang.org/packages/derelict-sdl2) | [decs](https://code.dlang.org/packages/decs)

Similar to e.g.
![image](https://user-images.githubusercontent.com/288976/35439628-cdbad1a0-029b-11e8-9102-7ca52a54702c.png)